views/authentication/home.hbs

```diff
@@ -27,8 +27,42 @@
 	</div>
 
 	<!-- GHOST (will be replaced by js) -->
-	<div class="sc-blog">
-    
+    <div class="sc-blog">
+        <div class="content" style="opacity: 1; transition: opacity 0.7s ease-in 0s;">
+			<div class="container">
+				<h2 class="mb-3 section-heading">Das macht die HPI Schul-Cloud aus</h2>
+				<div class="row section flex-container section-landing">
+					<div class="col-sm-12 col-md-6">
+						<img class="mb-3 mb-md-0" src="https://s3.hidrive.strato.com/schul-cloud-hpi/default/images/product.png" alt="">
+					</div>
+					<div class="col-sm-12 col-md-6">
+						<div class="pull-right">
+							<div class="flex-container mb-1">
+								<img class="check" src="https://s3.hidrive.strato.com/schul-cloud-hpi/default/images/check-icon-1.svg">
+								<p class="ml-1 teaser">Gemeinsam lernen in digitalen Räumen</p>
+							</div>
+							<div class="flex-container mb-1">
+								<img class="check" src="https://s3.hidrive.strato.com/schul-cloud-hpi/default/images/check-icon-1.svg">
+								<p class="ml-1 teaser">Lernmaterial, Aufgaben, Chat, Videokonferenzen und mehr</p>
+							</div>
+							<div class="flex-container mb-1">
+								<img class="check" src="https://s3.hidrive.strato.com/schul-cloud-hpi/default/images/check-icon-1.svg">
+								<p class="ml-1 teaser">Kostenlos für alle Schulen</p>
+							</div>
+							<div class="text-xs-center text-md-left">
+								<a class="btn btn-primary mt-1" href="mailto:info@schul-cloud.org?subject=Test-Account für die HPI Schul-Cloud
+								&body=Liebes HPI Schul-Cloud-Team,%0D%0Abitte senden Sie mir einen 14 Tage gültigen Testaccount kostenlos und unverbindlich zu.%0D%0A %0D%0A(bitte ausfüllen):%0D%0AVorname und Nachname:%0D%0AE-Mail-Adresse:%0D%0ASchule oder Organisation:%0D%0A %0D%0ARolle in dieser Schule oder Organisation:%0D%0A___________________________________________%0D%0A %0D%0AWas ist der HPI Schul-Cloud Testaccount?%0D%0A %0D%0AMit der HPI Schul-Cloud können Sie Ihren Unterricht datenschutzkonform in einem virtuellen Klassenzimmer gestalten. Sie können beispielsweise Videokonferenzen abhalten, Hausaufgaben digital aufgeben und abgeben, interaktive Unterrichtstools einbinden oder Lernmaterialien sicher recherchieren.%0D%0ADer 14 Tage gültige, anonymisierte Testaccount gibt Ihnen Gelegenheit, die HPI Schul-Cloud selbst zu erkunden. Ihnen werden die Zugangsdaten werktags so schnell wie möglich an die angegebene E-Mail-Adresse zugesandt. Nach 14 Tagen wird Ihr Testaccount und alle darin enthaltenden Daten automatisch gelöscht, weitere Schritte von Ihrer Seite sind nicht erforderlich.%0D%0AVielen Dank!">
```

Comment:
I have never tried this on other sites, but does the user understand what to do? Since this has already been discussed with Mona, I assume that the answer is yes, right? 

As soon as I get the go from whoever it is, I will put the code into Ghost